website/docs/r/compute_instance.html.markdown

```diff
@@ -272,3 +272,13 @@ exported:
 * `disk.0.disk_encryption_key_sha256` - The [RFC 4648 base64](https://tools.ietf.org/html/rfc4648#section-4)
     encoded SHA-256 hash of the [customer-supplied encryption key]
     (https://cloud.google.com/compute/docs/disks/customer-supplied-encryption) that protects this resource.
+
+## Import
+
+~> **Note:** The fields `boot_disk.0.initialize_params`, `boot_disk.0.disk_entryption_raw` and `attached_disk.*.disk_encryption_key_raw` cannot be imported automatically. The API doesn't return this information. You will need to update your state manually for this field after importing the resource.
```

Comment:
Just to make it extra clear, I'd say something like "If you are setting one of these fields in your config, you will need to update your state..." since not everybody is going to need to update their state.

Reply:
Good point. Will do